components/apps_form/apps_form_component.css

```diff
@@ -0,0 +1,3 @@
+.modal-body-opacity {
+  opacity: .1 !important;
```

Comment:
I honestly can't see any content in the modal when it is showing 10% opacity (Maybe I'm blind 🙂 )

Reply:
If you can make this available to a UX designer, they'll be able to modify the css within the browser, and find the right balance and right approach to solve the problem within the context of the whole application.

You can create a cloud server with the commit using `/cloud create webapp-pr-8601 --version 1d4e29a`. Then you'll need to install and configure the Apps plugin with an App to test with. 

The reason the tag is `1d4e29a` is because it's the first 7 characters of the commit hash of your last commit `1d4e29a32a93104f71c4f1a226decd3e41d1b9b8`, which is how the docker tags are assigned to commits https://hub.docker.com/r/mattermost/mattermost-enterprise-edition/tags?name=1d4e29.

Notice on that page that it was uploaded to docker 7 hours ago, which is when you made your last commit on this PR. I think mattermost-server and mattermost-webapp CI is now creating docker images for each commit on the two repositories, which is awesome.

Reply:
FYI if you need to use an unreleased version of the zendesk app to test this (i.e. you need install an HTTP App), you won't be able to test this with the "Setup Cloud Test Server", because these servers are in "Cloud Mode" and don't support HTTP Apps.

Servers created with the `/cloud` command allow us to upload plugins, so we can upload a different proxy plugin with the CloudMode check disabled, and then install HTTP Apps.

Also to have a QA/staging environment for the Zendesk App, we can use Heroku. We can talk more about this offline